eogrow/core/schemas.py

```diff
@@ -47,6 +47,13 @@ class PipelineSchema(BaseSchema):
     workers: int = Field(
         1, description="Number of workers for parallel execution of workflows. Parameter does not affect ray clusters."
     )
+    worker_type: Optional[str] = Field(
```

Comment:
should this name be more specific to ray, since it's used exclusively for ray, or do we wish to keep it general in case we ever extend the functionality to other cluster platforms?

Reply:
i can add a `ray` prefix if you'd like
it's still short enough

Reply:
I feel it might be misleading otherwise.. idk. if we ever switch to resources, it may be easier to go from `ray_worker_type` to `ray_resources`

Reply:
done